### PR TITLE
Transactor private methods to plugin exports

### PR DESCRIPTION
**Pull request by oleks-rip**

### CODE REVIEW

File: src/ripple/plugin/exports.h [hunk 1]
```diff
@@ -88,6 +95,14 @@ struct TransactorExport
     checkPriorTxAndLastLedgerPtr checkPriorTxAndLastLedger = nullptr;
     checkFeePtr checkFee = nullptr;
     checkSignPtr checkSign = nullptr;
```
Comment (mvadari) [inline src/ripple/plugin/exports.h:101]: Can you also update the formatting in the rest of the struct?

**Review by mvadari — COMMENTED** [review]
One other note: the XLS-42d spec will also need to be updated with these new functions.

**Comment by mvadari** [comment]
lgtm other than the notes above and the clang format failed check

File: src/ripple/plugin/exports.h (hunk 1 above)
Comment (oleks-rip) [reply src/ripple/plugin/exports.h:101]: done

File: src/test/plugin/fixtures/TrustSet.cpp [hunk 2]
```diff
@@ -666,15 +931,21 @@ getTransactors()
...
          doApply,
-         NULL,
-         NULL,
-         NULL,
-         NULL}};
+         nullptr,
+         nullptr,
+         nullptr,
+         nullptr,
+         payFee,
```
Comment (mvadari) [inline src/test/plugin/fixtures/TrustSet.cpp:730]: Can these tests do more than just run the code? Ideally they would change something about these functions and make sure that that change is actually happening.

Comment (oleks-rip) [reply src/test/plugin/fixtures/TrustSet.cpp:730]: done 81c8243ba

File: src/test/plugin/fixtures/TrustSet.cpp [hunk 3]
```diff
@@ -666,24 +967,125 @@ getTransactors()
...
+         consumeSeqProxy,
+         preCompute,
+         apply,
+         checkSingleSign,
+         checkMultiSign},
```
Comment (mvadari) [inline src/test/plugin/fixtures/TrustSet.cpp]: You don't need these functions, you can just make them `nullptr`

File: src/test/plugin/Plugins_test.cpp [hunk 4]
```diff
@@ -605,6 +605,169 @@ class Plugins_test : public beast::unit_test::suite
...
+            {
+                auto& ja = jv[jss::LimitAmount] =
+                    USD(1000).value().getJson(JsonOptions::none);
+                ja[jss::issuer] = bob.human();
+            }
+            auto const& tx = env.jt(jv);
+
+            Serializer s;
+            tx.stx->add(s);
+            auto const& jr = env.rpc("submit", strHex(s.slice()));
```
Comment (mvadari) [inline src/test/plugin/Plugins_test.cpp]: You can submit the transaction with just `env(jv);` instead of needing this.

Comment (oleks-rip) [reply src/test/plugin/Plugins_test.cpp]: I need to check response, and I didn't found other way to get it. Is it possible?

Comment (mvadari) [reply src/test/plugin/Plugins_test.cpp]: You can do that with e.g. `env(jv, ter(tecINTERNAL));`

File: src/test/plugin/fixtures/TrustSet.cpp (hunk 3 above)
Comment (oleks-rip) [reply src/test/plugin/fixtures/TrustSet.cpp]: done 3de83f0c1e7e9

File: src/test/plugin/Plugins_test.cpp [hunk 5]
```diff
@@ -605,6 +605,169 @@ class Plugins_test : public beast::unit_test::suite
...
+            auto const jr = env.rpc("submit", strHex(s.slice()));
+
+            if (BEAST_EXPECT(
+                    jr.isObject() && jr.isMember(jss::result) &&
+                    jr[jss::result].isMember(jss::engine_result_code)))
+            {
+                BEAST_EXPECT(
+                    jr[jss::result][jss::engine_result_code].asInt() ==
+                    tecINTERNAL);
+            }
```
Comment (mvadari) [inline src/test/plugin/Plugins_test.cpp:659]:
```suggestion
            env(jv, tecINTERNAL);
```

Comment (mvadari) [reply src/test/plugin/Plugins_test.cpp:659]: ditto for all the other tests

File: src/test/plugin/fixtures/TrustSet.cpp [hunk 6]
```diff
@@ -650,6 +652,56 @@ doApply(ApplyContext& ctx, XRPAmount mPriorBalance, XRPAmount mSourceBalance)
     return terResult;
 }
 
+TER
+payFee(ApplyContext& ctx, XRPAmount& sourceBalance)
```
Comment (mvadari) [inline src/test/plugin/fixtures/TrustSet.cpp:656]: This function can also be removed.

**Pull request closed without merging**